src/hooks/useCurrencyLogoURIs.ts

```diff
@@ -36,6 +42,13 @@ function getTokenLogoURI(address: string, chainId: SupportedChainId = SupportedC
   if (networksWithUrls.includes(chainId)) {
     return `https://raw.githubusercontent.com/Uniswap/assets/master/blockchains/${networkName}/assets/${address}/logo.png`
   }
+
+  // Celo logo is hosted elsewhere.
+  if (isCelo(chainId)) {
+    if (address === nativeOnChain(chainId).wrapped.address) {
+      return 'https://raw.githubusercontent.com/ubeswap/default-token-list/master/assets/asset_CELO.png'
```

Comment:
Why is this hosted on ubeswap? Does celo have a canonical file that we can store and reference?

Reply:
Yea, this is copy pasta from the interface back when I didn't have a 'celo-org' hosted token list 
I'll replace it with 'https://celo-org.github.io/celo-token-list/assets/celo_logo.svg''